avm/res/compute/gallery/image/main.bicep

```diff
@@ -134,67 +39,61 @@ var builtInRoleNames = {
   )
 }
 
-resource gallery 'Microsoft.Compute/galleries@2022-03-03' existing = {
+resource gallery 'Microsoft.Compute/galleries@2023-07-03' existing = {
   name: galleryName
 }
 
-resource image 'Microsoft.Compute/galleries/images@2022-03-03' = {
+resource image 'Microsoft.Compute/galleries/images@2023-07-03' = {
   name: name
   parent: gallery
   location: location
-  tags: tags
+  tags: tags // use from the parameter, as the galleryImage.tags have been stored there in the gallery.bicep before calling the template
```

Comment:
Not sure I understand what the comment is telling me. But I do have the same feedback as for the update of it's parameter description 😄